core/src/test/java/com/novoda/noplayer/internal/exoplayer/ExoPlayerFacadeTest.java

```diff
@@ -206,6 +208,20 @@ public void selectSubtitleTrack_thenThrowsIllegalStateException() {
 
             facade.selectSubtitleTrack(subtitleTrack);
         }
+
+        @Test
+        public void whenSetVolume_thenThrowsIllegalStateException() {
```

Comment:
`whenSetVolumeWithoutLoadingVideoFirst_thenThrowsIllegalStateException`

Reply:
Same as above, the enclosing class is `GivenVideoNotLoaded`